CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ option(UMF_BUILD_LIBUMF_POOL_JEMALLOC "Build the libumf_pool_jemalloc static lib
 option(UMF_BUILD_LIBUMF_POOL_SCALABLE "Build the libumf_pool_scalable static library" OFF)
 option(UMF_BUILD_TESTS "Build UMF tests" ON)
 option(UMF_BUILD_BENCHMARKS "Build UMF benchmarks" OFF)
+option(UMF_BUILD_BENCHMARKS_MT "Build UMF multithreaded benchmarks" OFF)
```

Comment:
See line no. 118:
```
if(UMF_BUILD_BENCHMARKS)
```
Shouldn't it be:
```
if(UMF_BUILD_BENCHMARKS OR UMF_BUILD_BENCHMARKS_MT)
```
?

Reply:
I've made it dependent on the UMF_BUILD_BENCHMARKS flag for now.